### PR TITLE
Add integration tests for prepare and submit flow

### PR DESCRIPTION
**Pull request by clark800**

### CODE REVIEW

File: test/integration/integration-test.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+var supertest = require('supertest');
+var wallet = require('./wallet');
+var fixtures = require('../fixtures');
+
+var PREFIX = '/v1';
+var TIMEOUT = 20000;   // how long before each test case times out
+var INTERVAL = 2000;   // how long to wait between checks for validated ledger
+var API = supertest('http://localhost:5990');
+
+
```
Comment (geertweening) [inline test/integration/integration-test.js:13]: extra linebreak?

File: test/integration/integration-test.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  assert(body && body.transaction);
+  assert.strictEqual(body.transaction.Account, wallet.getAddress());
+  assert.strictEqual(body.transaction.TransactionType, transactionType);
+  assert.strictEqual(body.transaction.meta.TransactionResult, 'tesSUCCESS');
+}
+
+suite.skip('integration tests', function() {
+
+  test('settings', function(done) {
+    this.timeout(TIMEOUT);
```
Comment (geertweening) [inline test/integration/integration-test.js]: can't you set timeout for the entire suite instead of duplicating for each test?

**Comment by geertweening** [comment]
besides open comments, LGTM

File: test/integration/integration-test.js (hunk 1 above)
Comment (clark800) [inline test/integration/integration-test.js:13]: Intentional, the style guide currently allows up to two consecutive blank lines.

File: test/integration/integration-test.js (hunk 2 above)
Comment (clark800) [inline test/integration/integration-test.js]: Looks like it does allow it on the suite, I'll try it.

**Pull request merged**